pwnlib/tubes/ssh.py

```diff
@@ -1090,6 +1092,8 @@ def is_exe(path):
                 h.failure("Process creation failed")
                 self.warn_once('Could not find a Python interpreter on %s\n' % self.host \
                                + "Use ssh.run() instead of ssh.process()")
+                if python.connected():
+                    python.close()
```

Comment:
We can just call `python.close()` without checking `connected`

Reply:
Cool, will do. Even my experience says it's better to do a check before just close()

Reply:
I'm pretty sure you can call `io.close(); io.close(); io.close();` as many times as you want and nothing bad happens